handlers/organizations.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const Formatters = require('../lib/formatters');
+
+function errorHandler(reply, e) {
+  return this.helpers.errorHandler.apply(this, arguments);
+}
+
+// GET /organizations
+exports.listOrganizations = async function({ auth, query}, reply) {
+  try {
+    const { userOrganizationRoleId, super: superUser } = auth.credentials;
+    
```

Comment:
@fdvj I know.. i need manage who see the list of organizations (problem linked below in test/functional/organizations.js)

Reply:
For the 403, this is not necessary. I've added an option to the policy plugin that validates if the user is super or not, and returns a 403 according.

In the route file, just above the `hapi-swagger` plugin, you can use the policy plugin, like this:

```javascript
// GET /organizations
routes.push({
  method: 'GET',
  path: API_BASE_PATH,
  config: {
    handler: Organizations.listOrganizations,
    description: 'List all organizations',
    notes: 'List all organizations in the system',
    plugins: {
      'policy': {
        super: true
      },
      'hapi-swagger': {
        responses: {
          '200': {
            description: 'Success',
            schema: SCHEMAS.Organizations
          },
          '401': {
            description: 'Unauthorized',
            schema: SCHEMAS.Errors.AuthenticationError
          },
          '403': {
            description: 'Forbidden',
            schema: SCHEMAS.Errors.ForbiddenError
          },
          '500': {
            description: 'Internal Server Error',
            schema: SCHEMAS.Errors.InternalServerError
          }
        }
      }
    },
    tags: ['api']
  }
});
```

With this, if the user is not super, it will automatically throw a 403